server/app/views/questiontypes/IdQuestionFragment.html

```diff
@@ -0,0 +1,21 @@
+<div th:fragment="id(question)"
+     th:with="idQuestion=${question.createIdQuestion()},
+              questionId=${#strings.randomAlphanumeric(8)},
+              inputId=${#strings.randomAlphanumeric(8)}"
+     th:id="${questionId}"
+     class="cf-question-text">
+    <!-- Title and Help Text -->
+    <div th:replace="~{questiontypes/QuestionBaseFragment :: titleAndHelpTextSingleInput(${question}, ${questionId}, ${inputId})}"></div>
+
+    <!-- Display errors -->
+    <div th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${idQuestion.getValidationErrors().get(question.getContextualizedPath())})}"></div>
+    
+    <input type="text"
```

Comment:
question (blocking):  In `IDQuestionDefinition.java` I see min and max lengths for the input.  Should those be added as attributes here?

Reply:
The min and max input lengths are enforced on the server side and not added to the dom. I did miss the maxlength=10000 that's added to all inputs, so i added that

Reply:
I see.  I think it would be useful (and easy) to add client side validation using the native min and max length attributes.  That would have the big benefit of providing some immediate feedback to the user.  Would you consider adding this or do you think that it should be separate work?

Reply:
I created an issue to chat with Sijia about this under the north star epic https://github.com/civiform/civiform/issues/7001 

After doing a bit of research, I think that minlength is a net negative because it basically gives a worse version of our error message (see screenshot in issue), and I could imagine it being confusing to use maxlength but not minlength. So moral of the story I'd prefer to punt from this PR and chat with Sijia about